source/_docs/mqtt/discovery.markdown

```diff
@@ -192,6 +195,10 @@ Supported abbreviations:
     'pow_cmd_t':           'power_command_topic',
     'pow_stat_t':          'power_state_topic',
     'pow_stat_tpl':        'power_state_template',
+    'pr_mode_cmd_t':       'preset_mode_command_topic',
+    'pr_mode_stat_t':      'preset_mode_state_topic',
+    'pr_mode_value_tpl':   'preset_mode_value_template',
```

Comment:
'pr_mode_value_tpl' should be 'pr_mode_val_tpl'

This was already checked in abbreviations.py

Reply:
Commited the fix